src/bun.js/bindings/bindings.zig

```diff
@@ -6017,8 +6032,8 @@ pub const CallFrame = opaque {
 
     fn Arguments(comptime max: usize) type {
         return struct {
-            ptr: [max]JSC.JSValue,
-            len: usize,
+            ptr: [max]JSC.JSValue = .{.undefined} ** max,
```

Comment:
This initializes it twice. It's already zeroed in `init`.

Reply:
i changed this impl so the elements default to `.undefined` instead of `.zero`